OpenRA.sln

```diff
@@ -30,6 +30,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tiberian Dawn Lua scripts",
 		mods\cnc\maps\gdi05b\gdi05b.lua = mods\cnc\maps\gdi05b\gdi05b.lua
 		mods\cnc\maps\gdi06\gdi06.lua = mods\cnc\maps\gdi06\gdi06.lua
 		mods\cnc\maps\gdi07\gdi07.lua = mods\cnc\maps\gdi07\gdi07.lua
+		mods\cnc\maps\gdi09\gdi09.lua = mods\cnc\maps\gdi09\gdi09.lua
+		mods\cnc\maps\gdi09\gdi09-AI.lua = mods\cnc\maps\gdi09\gdi09-AI.lua
```

Comment:
Actually, the `-AI` script should be placed first, but this is not really an issue.